lib/screens/v2/widget_instance/cr_departures.ex

```diff
@@ -47,10 +47,10 @@ defmodule Screens.V2.WidgetInstance.CRDepartures do
         }) do
       %{
         departures:
-          Enum.map(
-            departures_data,
-            &CRDeparturesWidget.serialize_departure(&1, config.wayfinding_arrows)
-          ),
+          departures_data
+          |> Enum.map(&CRDeparturesWidget.serialize_departure(&1, config.wayfinding_arrows))
+          |> Enum.slice(0..2),
+        show_via_headsigns_message: config.show_via_headsigns_message,
```

Comment:
Instead of doing this twice, `audio_serialize` could just use `serialize`. Looks like they both return the same thing.